---
{"dg-publish":true,"permalink":"/digital-garden/experiment1/","title":"Experiment I"}
---


# Experiment I: Molecular Cloning of Bacterial Inducible Plasmids

**Supervisors:** Alexander Geidies, Cha San Koh, Maria Medar, Sara Shumka, Daniel Wiebensohn

alexander.geidies@hhu.de, koh@hhu.de, marija.medar@hhu.de, sara.shumka@uni-duesseldorf.de, daniel.wiebensohn@uni-duesseldorf.de

Constructing recombinant genes within plasmids facilitates their recovery, genetic engineering *ex-vivo*, and shuttling into different desired strains or hosts. Recombinant DNA technologies represent the foundation for generating constitutive or inducible expression systems for the study of heterologous genes in different systems of interest.

In this first experimental part, we will design the required vectors for Experiment III, the design of protein logic gates based on protein *trans*-splicing in bacteria. For this purpose, we will use two different molecular DNA cloning techniques: 

A. Restriction and ligation cloning. This technique relies on the presence of restriction enzyme (dE) recognition sites flanking the desired insert and as well as at the desired positions in the recipient plasmid. The REs will then generate double DNA cuts leaving cohesive or blunt ends in both DNA sequences, in two independent reactions. Afterwards, we will purify the digested fragments and ligate them together using an enzyme called T4 DNA ligase, before transformation into chemically competent *E. coli* TOP10 bacteria for subsequent amplification and selection. T4 DNA ligase will conduct the repair of the broken DNA by forming a phosphodiester bond between a nearby 5’‑phosphate and 3’-OH ends resulting in the assembly of the desired new vector (**Figure 12**).

B. Advanced Quick Assembly (AQUA) Cloning. In contrast to conventional restriction/ligation cloning, this technique allows us to design a vector independently of the presence of suitable RE sites or the use of DNA restriction enzymes and ligases. This versatile and inexpensive technique relies on *in-vivo* DNA recombination within *E. coli* and therefore, requires DNA fragments that share an overlapping region of 20-60 bp at the junction sites. The homology region can for example be generated by PCR using appropriate primers that add these sequences to the products (**Figure 12**). AQUA Cloning can also be used to clone multiple DNA fragments simultaneously, reminiscent to other “DNA assembly cloning” methods. Additionally, a combination of restriction of the backbone and PCR amplification of the insert (as we will see in our exercise) can be used.

Each group will perform one ligation and one AQUA cloning experiment from one of the four plasmids:

- Groups 1 and 2: pDD259.
- Groups 3 and 4: pDD260.
- Groups 5 and 6: pDD261.
- Groups 7 and 8: pDD262.

As outlined in **Figure 12**, we will first perform a PCR reaction to amplify: a. The gene encoding the fluorescent protein mCherry interrupted by the full-length *Npu*DnaE intein gene (Int); b. The mCherry gene was disrupted by a mutated (non-functional) version of the intein; c. A DNA fragment encoding an N-terminal portion of mCherry (mChN) fused to an N-terminal split-fragment of the intein (IntN); and d. a DNA fragment encoding a C-terminal portion of mCherry (mChC) fused to a C-terminal split-fragment of the intein (IntC). The destination vectors will be digested overnight with the annotated restriction enzymes to generate the so-called vector “backbones” (this step will be performed by the supervisors in advance).

After gel purification of the PCR products, we will need to digest the PCR product with the same enzymes as the destination vectors in order to clone the genes via ligation cloning. We will then prepare a pre-mix with a 3:1 ratio of the corresponding fragments (insert:backbone). For the restriction/ligation protocol, remember to add T4 DNA ligase and the corresponding buffer! After ligation, the DNA mixtures will be transformed into competent Top10 *E. coli* cells and plated on LB agar plates containing the appropriate antibiotics for selection of transformed cells.

The engineered vectors will be purified and analyzed using analytical restriction enzyme digestion and agarose gel electrophoresis. We encourage you to consider beforehand which enzymes we could use to differentiate the restriction pattern expected for the original plasmids and that resulting from the digestion of the engineered vectors.

Finally, we will select some promising samples and analyze them by Sanger sequencing.

![Figure 13. Illustration of the cloning strategy. Top, schematic representation of the cloned vectors. Bottom, pRSFDuet-1 plasmid (KanR) digested with BamHI and KpnI serves as backbone for cloning plasmids pDD259/262 and 260. The inserts for these constructs are the amplification products of pDD002 and pDD083 with the primers oDD561 and oDD562; or the PCR product of pDD002 with oDD561 and oDD563, respectively.](/img/user/figures/cloning_strategies.png)

A third insert amplified from pDD002 with the primer oDD564 and oDD565 will be cloned into pHBBAD090 backbone, previously digested with *Nde*I and *Hin*dIII, to obtain pDD261. The primers were designed so that the PCR products possess the required homology regions to the digested vector backbones for AQUA Cloning. Alternatively, the PCR products can be digested using the indicated enzymes for ligation cloning. Both approaches result in the same final vectors.The \* symbol represents the location of the C1A mutation in the pDD262.

### Materials

| Category                    | Item                                   | Composition                                                                                | Conc.         | Use                           |
|-----------------------------|----------------------------------------|--------------------------------------------------------------------------------------------|---------------|-------------------------------|
| **Plasmids**                | pRSFDuet-1                             | PT7-H6-MCS-term (KanR)                                                                     |               | Digested backbone             |
|                             | pDD002                                 | PSV40-mCherryN-Int-mCherryC-pA (AmpR)                                                      |               | PCR template                  |
|                             | pHBBAD090                              | PBAD-Insert-term (AmpR)                                                                    |               | Digested backbone             |
|                             | pDD083                                 | PSV40-mCherryN-Int(C1A)-mCherryC-pA (AmpR)                                                 |               | PCR template                  |
| **Primers**                 | oDD561                                 | 5’-AGCAGCCATCACCATCATCACCACAGCCAG-GATCCGGTGAGCAAGGGCGAGGAG                                 | 25 µM         | mCh F *Bam*HI duet            |
|                             | oDD562                                 | 5’-GCAGCAGCGGTTTCTTTACCAGACTCGAGGG-TACCTTACTTGTACAGCTCGTCCATGC                             | 25 µM         | mCh R *Kpn*I duet             |
|                             | oDD563                                 | 5’-GCAGCAGCGGTTTCTTTACCAGACTCGAGGG-TACCTTAATTCGGCAAATTATCAACCCGC                           | 25 µM         | NpuEN R *Kpn*I duet           |
|                             | oDD564                                 | 5’-TGTTTAACTTTAAGAAGGAGATATACATATG-ATCAAATAGCCACACGTAAATATTTAGG                            | 25 µM         | NpuE\_C F *Nde*I bad          |
|                             | oDD565                                 | 5’-AAAATCTTCTCTCATCCGCCAAAACAGCCA-AGCTTACTTGTACAGCTCGTCCATGC                               | 25 µM         | mCh R *Hind*III bad           |
| **PCR reaction**            | Q5® High-Fidelity DNA polymerase (NEB) |                                                                                            | 2,000 U/mL    |                               |
|                             | Q5 Reaction buffer (5x)                | 2 mM MgCl$_2$                                                                              | 5X            |                               |
|                             | dNTPs                                  | dATP, dCTP, dGTP and dTTP                                                                  | 10 mM each    |                               |
| **Digestion reaction**      | Cutsmart (10x) (NEB)                   | 50 mM potassium acetate, 20 mM Tris-acetate, 10 mM magnesium acetate, 100 μg/mL BSA        | 10X           |                               |
|                             | Restriction enzymes (NEB)              | selected during the course                                                                 | 3,333 U/mL    |                               |
| **Gel electrophoresis**     | Agarose                                |                                                                                            |               |                               |
|                             | TE Buffer                              | 10 mM Tris-Cl pH 7.5, 1 mM EDTA                                                            |               |                               |
|                             | Gelred                                 |                                                                                            |               | Intercalation agent           |
|                             | Loading buffer                         | 50% Glycerol, 100 mM Na$_2$EDTA, pH 8.0, 1% SDS, 0.1% Bromophenol Blue, 0.1% Xylene Cyanol | 6X            |                               |
|                             | 1 Kb DNA ladder                        |                                                                                            |               | Ladder for 100 bp up to 10 kb |
| **M.-Nagel Gel extraction** | NT1 Buffer                             |                                                                                            |               | Binding buffer                |
|                             | NT3 Buffer                             |                                                                                            |               | Wash buffer                   |
|                             | M&N spin columns                       |                                                                                            |               | DNA binding                   |
| **Ligation**                | T4 Ligase Buffer (NEB)                 | 300 mM Tris-HCl (pH 7.8), 100 mM MgCl$_2$, 100 mM DTT, 10 mM ATP                           | 10X           |                               |
|                             | T4 DNA-ligase (NEB)                    |                                                                                            | 400,000 U/mL  |                               |
| **Wizard mini-prep kit**    | W1                                     | 50 mM Tris-HCl (pH 7.5), 10 mM EDTA, 100 μg/mL RNase A                                     |               | Resuspension solution         |
|                             | W2                                     | 0.2 M NaOH, 1% SDS                                                                         |               | Lysis solution                |
|                             | W3                                     | 4.09 M guanidine hydrochloride, 0.759 M potassium acetate, 2.12 M glacial acetic acid      |               | Neutralization solution       |
|                             | W4                                     | 60% ethanol, 60 mM potassium acetate, 8.3 mM Tris-HCl, 0.04 mM EDTA                        |               | Wash solution                 |
|                             | Wizard columns                         |                                                                                            |               | DNA purification              |
| **Others**                  | ddH$_2$O                               |                                                                                            |               |                               |
|                             | LB media                               |                                                                                            |               | Bacteria growth media         |
|                             | LB-plates with ampicillin              |                                                                                            | 100 μg/mL Amp |                               |
|                             | LB-plates with kanamycin               |                                                                                            | 50 μg/mL Kan  |                               |
|                             | Top10 Invitrogen™                      | Chemically competent *E. coli* cells                                                       |               |                               |

### Protocols

#### Day 1

##### 1.1 PCR

**Table 4.** PCR mixture.

| Component          | Amount | Unit   |
|--------------------|--------|--------|
| Q5 Reaction Buffer | 10     | µL     |
| Primer 1, 25 µM    | 1      | µL     |
| Primer 2, 25 µM    | 1      | µL     |
| dNTPs, 10 mM       | 1      | µL     |
| Template DNA       | 1      | µL     |
| Q5 DNA polymerase  | 0.5    | µL     |
| add dH$_2$O        | 32.5   | µL     |
| **Total**          | **50** | **µL** |

**Table 5.** Primers and templates.

| PCR             | Forward primer | Reverse primer | Template | Product size (bp) | Ta  |
|-----------------|----------------|----------------|----------|-------------------|-----|
| PCR259 (pDD259) | oDD561         | oDD562         | pDD002   | ca. 1190          | TBD |
| PCR262 (pDD262) | oDD561         | oDD562         | pDD083   | ca. 1190          | TBD |
| PCR260 (pDD260) | oDD561         | oDD563         | pDD002   | ca. 803           | TBD |
| PCR261 (pDD261) | oDD564         | oDD565         | pDD002   | ca. 454           | TBD |

**Table 6.** PCR program

| Temperature | Time    | Cycles |     |
|-------------|---------|--------|-----|
| 98°C        | 5 min   | 1×     |     |
| 98°C        | 30 s    | 25×    |     |
| Ta°C        | 30 s    | 25×    |     |
| 72°C        | 40 s/kb | 25×    |     |
| 72°C        | 10 min  | 1×     |     |
| 4°C         | Hold    | \-     |     |

##### 1.2 DNA Purification from Agarose Gel

1.  Agarose-gel electrophoresis: 0.8% agarose in TE Buffer, 1:10,000 Gelred Stock solution (Caution! DNA dyes intercalate, thus are mutagenic agents. Handle it with care).
2.  Sample preparation: Mix 50 µL plasmid DNA with 10 µL of 6x loading buffer.
3.  Use 12 µL DNA-Ladder (MM).
4.  Run gel at 120 V for 20–30 min.

![Figure 14. GeneRuler™ 1 kb DNA Ladder (Fermentas)](/img/user/figures/dna_ladder.png)

##### 1.3 DNA Purification

1.  Cut bands from the gel and transfer each gel slice into a 1.5 mL tube.
2.  Add approx. 200 µL NTI buffer / 100 mg gel and incubate the tube at 60°C, 200 rpm for \~ 10 min until the gel slice is completely dissolved.
3.  Add the samples onto M&N spin columns, 1 column per gel slice.
4.  Centrifuge at max speed for 60 s at RT and discard the flow through.
5.  Apply remaining sample volumes (if any).
6.  Centrifuge at max speed for 60 s at RT and discard flow-through.
7.  Add 700 µL NT3 buffer to the spin column.
8.  Centrifuge at max speed for 60 s at RT and discard flow through.
9.  Add 700 µL of NT3 buffer to the spin column as an additional washing step.
10. Centrifuge at max speed for 60 s at RT and discard flow through.
11. Centrifuge at max speed for 60 s at RT again and discard flow-through! (To dry the column and remove residual EtOH).
12. Place the column into a fresh and labelled 1.5 mL tube.
13. Add 20 µL ddH~2~O to the column
14. Incubate for 5 min at 70°C.
15. Centrifuge at max speed for 90 s at RT.
16. **KEEP** the flow through, it contains your purified DNA, determine the concentration using the Nanodrop.
17. Store at -20°C.

##### 1.4 Backbone Digestion (Performed by the Supervisors)

**Table 7.** Digestion of vector backbone.

| Component                            | Volume    |
|--------------------------------------|-----------|
| Plasmid DNA (2 µg)                   | x µL      |
| Cut Smart buffer (10x)               | 5 µL      |
| Enzyme 1                             | 2.5 µL    |
| Enzyme 2                             | 2.5 µL    |
| CIP (only for the ligation backbone) | 1 µL      |
| H$_2$O                               | x µL      |
| **Total**                            | **50 µL** |

- Incubate ON at 37°C.

We will provide you with the digested destination vectors.

##### 1.5 PCR Products Digestion (Only for Ligation Cloning)

This step will only be performed by the group member doing ligation cloning.

**Table 8.** Digestion of PCR products.

| Component              | Volume    |
|------------------------|-----------|
| PCR product            | 23 µL     |
| Cut Smart buffer (10x) | 3 µL      |
| Enzyme 1               | 2 µL      |
| Enzyme 2               | 2 µL      |
| H$_2$O                 | 0 µL      |
| **Total**              | **30 µL** |

- Incubate ON at 37°C.

#### Day 2

##### 2.1 Digested PCR Products Purification (Only for Ligation Cloning)

1.  Add approx. 60 µL NTI buffer.
2.  Add the samples onto M&N spin columns, 1 column per digestion.
3.  Centrifuge at max speed for 60 s at RT and discard the flow through.
4.  Apply remaining sample volumes (if any).
5.  Centrifuge at max speed for 60 s at RT and discard flow-through.
6.  Add 700 µL NT3 buffer to the spin column.
7.  Centrifuge at max speed for 60 s at RT and discard flow through.
8.  Add 700 µL of NT3 buffer to the spin column as an additional washing step.
9.  Centrifuge at max speed for 60 s at RT and discard flow through.
10. Centrifuge at max speed for 60 s at RT again and discard flow through! (To dry the column and remove residual EtOH).
11. Place the column into a fresh and labelled 1.5 mL tube.
12. Add 20 µL ddH~2~O to the column
13. Incubate 5 min at 70°C.
14. Centrifuge at max speed for 90 s at RT.
15. **KEEP** the flow through, it contains your purified DNA, determine the concentration using the Nanodrop.

##### 2.2 Assembly of the New Vector

###### 2.2.1 AQUA Cloning

If possible, use approximately 35 ng of DNA for the backbone.

Calculations for the amount of PCR-DNA needed for a 1:3 ratio:

**Table 9.** **Sample 1, Sample 2 (negative control)**

| Component         | Volume    |
|-------------------|-----------|
| Digested backbone | x µL      |
| PCR product       | x µL      |
| H$_2$O            | x µL      |
| **Total**         | **10 µL** |

- Incubate at RT for 60 min.
- Put samples on ice, 5 min.

###### 2.2.2 Ligation Protocol

**Table 10.** DNA ligation

| Component                | Volume    |
|--------------------------|-----------|
| Vector (30-50 ng)        | x µL      |
| Insert (3x excess)       | y µL      |
| H$_2$O (qsf 20 µL)       | z µL      |
| T4 DNA ligase Buffer 10x | 2 µL      |
| T4 DNA ligase            | 1 µL      |
| **Total**                | **20 µL** |

- Incubate at RT for 60 min.
- Heat inactivate the T4 DNA ligase by incubating 10 min at 65 °C.

##### 2.3 Transformation into *E. coli* Competent Cells

1.  AQUA/Ligation mixture and Negative control mixture
2.  Thaw cells on ice. Carefully add AQUA mix/ negative control (10 µL) or ligation mix/negative control (5 µL). Do not resuspend, just slightly stir using the pipette tip.
3.  Incubate for 30 min on ice, heat-shock for 45 s at 42°C, place on ice for 2 min, add 250 µL LB‑medium and incubate for 1 h at 37°C, 550 rpm.
4.  Finally, plate cells on ampicillin (100 µg/mL) or kanamycin (50 µg/mL) LBagar plates. For the AQUA plate 250 µL/plate and for ligation 100 µL/plate. (In this way, one may ensure the convenient isolation of colonies the next day).
5.  Distribute cells uniformly on the plates.
6.  Incubate plates overnight at 37°C.

#### Day 3

##### 3.1 Streaking Single Colonies on New LB-Ampicillin Plates

- Pick 4 single clones and streak out onto LB plates with ampicillin (100 μg/mL) or kanamycin (50 μg/mL). Streak 2 colonies in each plate. Incubate overnight at 37°C. Make sure to evenly spread the picked colony over the **whole surface** of the LB agar plate.

#### Day 4

##### 4.1 Plasmid DNA Isolation

1.  Using a sterile inoculating loop, scrape bacteria off the plate and resuspend in 250 μL of Cell Resuspension Solution (W1).
2.  Add 250 μL of Cell Lysis Solution (W2) to each sample; **invert** 4 times to mix.
3.  Incubate for 5 min at RT.
4.  Add 350 μL of Neutralization Solution (W3); **invert** 4–6 times to mix.
5.  Centrifuge at max speed for 8 min at RT.
6.  During the centrifugation insert spin columns into collection tubes.
7.  Equilibrate columns with 700 µL ddH~2~O, centrifuge for 60 s at max speed and discard the flow‑through.
8.  After centrifugation, transfer cleared lysates into spin columns.
9.  Centrifuge at max speed for 60 s at RT.
10. Discard flow-through and reinsert columns into collection tubes.
11. Add 750 μl of Wash Solution (WW). Centrifuge at top speed for 60 s. Discard flow-through and reinsert the column into the collection tube.
12. Repeat the previous step with 250 μl of Wash Solution (WW).
13. Centrifuge at max speed for 60 s at RT.
14. Transfer Spin Column into a sterile 1.5 mL tube.
15. Add 60 μl of prewarmed MilliQ-Water to the Spin Column and incubate for 4 min at RT. Centrifuge at max speed for 90 s at RT and **KEEP** the tube with the flow through.
16. Store extracted plasmids at -20°C.
17. Wash the mini column four times with 700 µl of ddH~2~O and keep them. Do not throw them away.

##### 4.2 Analytical Restriction Enzyme Digestion

For each clone, perform one digest of 5 µL as follows:

**Table 11.** Test digestion.

| Component              | Volume   |
|------------------------|----------|
| Plasmid DNA            | 4 µL     |
| Cut Smart buffer (10x) | 0.5 µL   |
| Enzyme 1               | 0.25 µL  |
| Enzyme 2               | 0.25 µL  |
| **Total**              | **5 µL** |

1.  Incubate at 37°C for 1–2 h.
2.  Analyze your digest via Agarose-gel electrophoresis: 1% agarose in TE Buffer, 1:10000 Gelred Stock solution (Caution! DNA dyes intercalate, thus are mutagenic agents. Handle it with care).
3.  Sample preparation: Mix 5 µL digest with 1–2 µL of 6× loading buffer.
4.  Run gel 120 V, 20–30 min. Perform graphical analysis of gel.

##### 4.3 Sequencing

Selection of positive colonies.

Send a 12 µL aliquot with a concentration of 40–100 ng/µL. Dilute if necessary.


---
[< Previous: Background](background.md) | [Next: Experiment II >](experiment2.md)
